.ci/azure-pipelines/main.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     workspace:
       clean: all
     pool:
-      vmImage: 'macOS-12'
+      vmImage: 'macOS-13'
```

Comment:
Imo you could also just upgrade to `macOS-14` so we can forget about it for a bit longer until it's eventually deprecated

Reply:
Agreed.